### ISSUE TITLE
confidence bands for learning curves of multiple runs

### ISSUE THREAD

**Issue by gfrogat** [Caleydo/malevo#74]
we should add error bands to the curves (see the function `tsplot` from the `seaborn` library --> https://seaborn.pydata.org/generated/seaborn.tsplot.html)

**Comment by thinkh** [comment]
Good idea. We should keep in mind that we need a (custom?) D3 implementation of this visualization.

**Comment by mstreit** [comment]
How about this?
http://bl.ocks.org/ndarville/6552457

**Comment by mstreit** [comment]
or this:
https://github.com/timaronan/d3-line-range

**Comment by thinkh** [comment]
Nice 👍

**Comment by gfrogat** [comment]
@thinkh @NMO13 @mstreit 

maybe we should consider this for the paper?

**Comment by mstreit** [comment]
yes, would be a nice addition

**Comment by NMO13** [comment]
In the detail view, it would make sense. But I think in the conf mat cells you wouldn't see it well enough.
